VimeoNetworking/VimeoNetworking.xcodeproj/project.pbxproj

```diff
@@ -14,6 +14,11 @@
 		01B5C8681CC826D800694F4A /* VimeoNetworking.h in Headers */ = {isa = PBXBuildFile; fileRef = 01B5C8671CC826D800694F4A /* VimeoNetworking.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		01B5C86F1CC826D800694F4A /* VimeoNetworking.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 01B5C8641CC826D800694F4A /* VimeoNetworking.framework */; };
 		01B5C8741CC826D800694F4A /* VimeoNetworkingTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 01B5C8731CC826D800694F4A /* VimeoNetworkingTests.swift */; };
+		01CDBEFE1CEBA2030093DDF4 /* VIMMappable.h in Headers */ = {isa = PBXBuildFile; fileRef = 01CDBEF91CEBA2030093DDF4 /* VIMMappable.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
Can you add a note to the top of the VIMObjectMapper repo pointing people to VimeoNetworking and the subspec?